recipes/clusterutils/meta.yaml

```
@@ -0,0 +1,47 @@
+{% set name = "clusterutils" %}
+{% set version = "0.3.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/choderalab/clusterutils/archive/{{ version }}.tar.gz
+  sha256: 2c841215d1a93af5c61fac2ff52cdc8a286f05af219bca84a76a7a7380f3ee96
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv --no-deps"
+  entry_points:
+  - build_mpirun_configfile = clusterutils.build_mpirun_configfile:main
+
+requirements:
+  host:
+    - python
```

(also applies to the other recipes)
```suggestion
    - python >=3.6
```